src/App.tsx

```diff
@@ -10,33 +12,25 @@ export const tabs = [
 ];
 
 export const App: React.FC = () => {
+  const [selectedTabId, setSelectedTabId] = useState('tab-1');
+
+  const onTabSelected = (tab: Tab | undefined) => {
```

Comment:
It is better to use the tab id as an argument for the onTabSelected function rather than the whole tab object.